analysis/parameter_tuning.py

```diff
@@ -153,6 +163,31 @@ def _find_candidates(histogram: histograms.Histogram) -> List:
         max_contributions_per_partition=linf_bounds)
 
 
+def _find_candidates_quantiles(histogram: histograms.Histogram) -> List:
+    QUANTILES_TO_USE = [0.9, 0.95, 0.98, 0.99, 0.995]
+    candidates = histogram.quantiles(QUANTILES_TO_USE)
+    candidates.append(histogram.max_value)
+    candidates = list(set(candidates))  # remove duplicates
+    candidates.sort()
+    return candidates
+
+
+def _find_candidates_constant_relative_step_func(n_max: int):
+
+    def _find_candidates_constant_relative_step(
+            histogram: histograms.Histogram) -> List:
+        max_value = histogram.max_value
+        # relative step varies from 1% to 0.1%
+        candidates = _generate_possible_contribution_bounds(max_value)
+        if len(candidates) > n_max:
+            candidates = candidates[::math.ceil(len(candidates) / n_max)]
+        if candidates[-1] != max_value:
+            candidates.append(max_value)
```

Comment:
Does it mean that it can be more than n_max elements? maybe candidates[-1] = max_value ?